packages/grid/_modules_/grid/GridComponent.tsx

```diff
@@ -220,7 +220,8 @@ export const GridComponent = React.forwardRef<HTMLDivElement, GridComponentProps
                       ref={columnsHeaderRef}
                       columns={internalColumns.visible || []}
                       hasScrollX={!!renderCtx?.hasScrollX}
-                      onResizeColumn={onResizeColumn}
+                      // @ts-expect-error fixed in v5
```

Comment:
Why do we have this one here? It can be fixed inside `useColumnResize` 
can we avoid `// @ts-expect-error ` as a practice please?